webapp/src/components/admin_settings/jitsi_section.tsx

```diff
@@ -0,0 +1,344 @@
+import * as React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+type Props = {
+    disabled: boolean,
+    onJitsiURLChange: any,
+    onJitsiEmbeddedChange: any,
+    onJitsiMeetingNamesChange: any,
+    onJitsiJwtAuthChange: any,
+    onJitsiAppIDChange: any,
+    onJitsiAppSecretChange: any,
+    onJitsiMeetingLinkExpChange: any,
+    onJitsiCompatibilityChange: any,
+    serverUrl: string,
+    embedded: boolean,
+    namingScheme: string,
+    jwtEnabled: boolean,
+    appID: string,
+    appSecret: string,
+    meetingLinkExpire: number,
+    compatibilityMode: boolean
+};
+
+export const JITSI_NAMING_SCHEME = {
+    WORDS: 'words',
+    UUID: 'uuid',
+    MATTERMOST: 'mattermost',
+    ASK: 'ask'
+};
+
+export class JitsiSection extends React.Component<Props> {
+    constructor(props: any) {
+        super(props);
+        console.log('JitsiSection');
+    }
+
+    render() {
+        // TODO rewrite common components or use Mattermost components
```

Comment:
You should also do this TODO in a separate PR.

Reply:
Will do